core/include/JellyfinQt/viewmodel/playbackmanager.h

```diff
@@ -98,42 +102,51 @@ class PlaybackManager : public QObject, public QQmlParserStatus {
     Q_PROPERTY(QString errorString READ errorString NOTIFY errorStringChanged)
     Q_PROPERTY(bool hasVideo READ hasVideo NOTIFY hasVideoChanged)
     Q_PROPERTY(bool seekable READ seekable NOTIFY seekableChanged)
-    Q_PROPERTY(QObject* mediaObject READ mediaObject NOTIFY mediaObjectChanged)
-    Q_PROPERTY(QMediaPlayer::MediaStatus mediaStatus READ mediaStatus NOTIFY mediaStatusChanged)
-    Q_PROPERTY(QMediaPlayer::State playbackState READ playbackState NOTIFY playbackStateChanged)
+    //Q_PROPERTY(QObject* mediaObject READ mediaObject NOTIFY mediaObjectChanged)
+    Q_PROPERTY(QObject* mediaObject READ player NOTIFY playerChanged);
```

Comment:
Keep property names/getters/notifiers consistent. Remove the commented out line above.